content/en/docs/concepts/security/index.md

```diff
@@ -545,11 +548,12 @@ an authorization policy to the workloads to enforce access control.
 For workloads without authorization policies applied, Istio doesn't enforce
 access control allowing all requests.
 
-Authorization policies support both `ALLOW` and `DENY` actions. The deny
-policies take precedence over allow policies. If any allow policies are applied
-to a workload, access to that workload is denied by default, unless explicitly
-allowed by the rule in the policy. When you apply multiple authorization
-policies to the same workload, Istio applies them additively.
+Authorization policies support `ALLOW`, `DENY` and `CUSTOM` actions and the evaluation order is `CUSTOM`, `DENY` and
```

Comment:
Can you separate this into two sentences? Evaluation order sounds like implementation details. Can we talk about the user visible impact (if not mentioned already)? e.g. policy precedence?

Reply:
here by "evaluation order" I actually mean the order of the action, which should be part of the API, the order of rules are implementation detail and not mentioned here. Updated to use the term "policy precedence".